### DISCUSSION TITLE
approved but no benefits

### DISCUSSION THREAD

**Discussion by SquishyGlue3** [community/community#120970] [GitHub Education]
### Hi everyone,

Hello, My application was approved 2 days ago but It seems I do not have the benefits, anyone knows why?

**Comment by gurjeetsinghvirdee** [comment]
Hey @SquishyGlue3 
It takes some time to reflect something around 1-2 weeks.
You can check your profile to see if your benefits are activated or not 
you will see a  `Purple PRO` badge in your profile like this: 
![image](https://github.com/community/community/assets/73753957/e296a7af-f1d1-4968-a016-2a87444c16a2)

It's a sign of that you are approved for GitHub Student Developer Pack

**Comment by SquishyGlue3** [comment]
![Screenshot 2024-04-24 at 22 53 00](https://github.com/community/community/assets/118686465/a3448bea-ef86-4d4a-a03d-af72dd2ce5a2)


It says here approved...

**Reply by gurjeetsinghvirdee** [reply]
Yeah, but it takes time to get your benefits now. Before, it would be received in 1-2 days, but now it takes much longer. 
You will receive another email similar to the one **welcoming you to the GitHub Student Developer Pack.**
